src/api/mock-data/dataplane-rules.json

```diff
@@ -0,0 +1,92 @@
+{
+  "total": 3,
+  "items": [
+    {
+      "type": "clientSubset",
+      "name": "192.168.0.1:80",
+      "service": "backend",
```

Comment:
@jakubdyszkiewicz is this actually possible? Do clientSubset actually have service?

Reply:
It's a service associated with an inbound 192.168.0.1:80 so we don't have to look for it in Dataplane object.

Reply:
In other words: this is a convenience field to save a lookup

Reply:
Ha so the `kuma.io/service` line 10 should be `backend` no?